packages/flutter/lib/src/material/drawer.dart

```diff
@@ -355,6 +356,11 @@ class DrawerController extends StatefulWidget {
   /// Optional callback that is called when a [Drawer] is opened or closed.
   final DrawerCallback? drawerCallback;
 
+  // If set to false, prevents the barrier behind the [Drawer] from being dismissed.
+  // Defaults to true. The value of this flag also gets inherited by the parent Scaffold,
+  // or can be overridden if a [DrawerController] is extended and this value set accordingly.
```

Comment:
This doc doesn't meet some of the [doc style guide](https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#dartdoc-specific-requirements). Following is my attempt of rewriting it: 
```suggestion
  /// Whether tapping the barrier behind the [Drawer] dismisses it.
  ///
  /// Defaults to true.
  ///
  /// If false, tapping the barrier will not dismiss the drawer.
```

I'm not sure if we should mention the other two sentences.

> The value of this flag also gets inherited by the parent Scaffold.

I don't quite understand this sentence, and I guess you meant to say "This setting is inherited from the parent [Scaffold]". I think this explanation belongs more to `Scaffold.drawDismissible`, since "who can set this field" isn't a property of this field.

> can be overridden if a [DrawerController] is extended

This doesn't need explanation either since it's the very use case of extending.

Reply:
Yes, I meant to say "This setting is inherited from the parent [Scaffold]".  On the other sentence, I wanted to be as explicit as possible and rather err on the side of caution as opposed to leave things to interpretation.